src/data/partnersData.tsx

```diff
@@ -0,0 +1,43 @@
+import Translate from "@docusaurus/Translate";
+import React from "react";
+
+import huawei from "../../static/img/supporters/huawei.png";
+import daocloud from "../../static/img/supporters/DaoCloud.png";
+import DSK from "../../static/img/supporters/DSK.png";
+
+const PartnersList = [
+  {
+    image: huawei,
+    title: "Huawei",
+    webURL: "https://www.huaweicloud.com/intl/en-us/",
+    description: (
+      <Translate id="partners.huawei.description">
+        Huawei Cloud is a leading cloud service provider in China and Asia
+        Pacific.
```

Comment:
```suggestion
        UCS provides consistent experience in cloud native application deployment, management, and ecosystem. Cloud native applications can freely run across regions and clouds with intelligent traffic distribution.
```
Get the introduction from https://www.huaweicloud.com/intl/en-us/product/ucs.html.

Reply:
Seems the description can not fully show out at the https://deploy-preview-531--karmada.netlify.app/partners/.